### PR TITLE
Events Timeline UI Fixes

### LINKED ISSUE #98: Fix ui, make it uing
image

### PR DESCRIPTION
**Pull request by divyanshu1810**
## UI
image

## closes #98

### CODE REVIEW

File: shared/components/events/event-timeline.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const EventTimeline = ({ event }: { event: EventType }) => {
               fontSizes={{
                 cardSubtitle: '1.3rem',
                 cardText: '0.9rem',
-                cardTitle: '1.2rem',
+                cardTitle: '1.0rem',
```
Comment (shawshankkumar) [inline shared/components/events/event-timeline.tsx:40]: have you tested it for iphone xr?

Comment (divyanshu1810) [reply shared/components/events/event-timeline.tsx:40]: yes all the screens!

Comment (divyanshu1810) [reply shared/components/events/event-timeline.tsx:40]: image

Comment (divyanshu1810) [reply shared/components/events/event-timeline.tsx:40]: 12 
image

Comment (shawshankkumar) [reply shared/components/events/event-timeline.tsx:40]: image

**Comment by shawshankkumar** [comment]
works for most screens. @utkarshini14 check this for major mobile sizes and let me know.

**Comment by utkarshini14** [comment]
works fine for all screens except for 'galaxy fold'
@shawshankkumar

**Review by shawshankkumar — APPROVED** [review]

**Pull request merged**